### PR TITLE
Scrapped rating graphs

### PR DESCRIPTION
**Pull request by karansdoshi**

### CODE REVIEW

File: Scrapping.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+username=input('Enter Username')
+page=requests.get("https://www.codechef.com/users/"+username)
+soup = BeautifulSoup(page.content, 'html.parser')
+#print(list(soup.children)[2])
+x=str(soup.select('script[type="text/javascript"]')[29]).split(';')[9][19:].strip()
```
Comment (Swapnilr1) [inline Scrapping.py]: I get IndexError: list index out of range when running this code. Don't hardcode this.

**Review by Swapnilr1 — CHANGES_REQUESTED** [review]

File: Scrapping.py (hunk 1 above)
Comment (karansdoshi) [reply Scrapping.py]: ok ill try to fix it

File: temp.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+username=input('Enter Username')
+page=requests.get("https://www.codechef.com/users/"+username)
```
Comment (Swapnilr1) [inline temp.py]: Add a browser user agent, or the scraper will be detected as a bot and blocked.
```suggestion
page=requests.get("https://www.codechef.com/users/"+username, headers={'User-Agent': 'Mozilla/5.0'})
```

File: Scrapping.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+username=input('Enter Username')
+page=requests.get("https://www.codechef.com/users/"+username)
```
Comment (Swapnilr1) [inline Scrapping.py]: Similar to temp.py. This is why I was getting IndexError due to CodeChef blocking this as a bot.
```suggestion
page=requests.get("https://www.codechef.com/users/"+username, headers={'User-Agent': 'Mozilla/5.0'})
```

**Review by Swapnilr1 — CHANGES_REQUESTED** [review]
This is almost OK. The entire scraping code should be a function with username as a parameter. You can choose which one of `Scrapping.py` or `temp.py` you want to keep (I tested after adding the browser user-agent and both work correctly).
This will require further changes when I want to extract and process the data for training, but for now good work.

**Pull request closed without merging**